### DISCUSSION TITLE
Me ajudem a escolher um notebook

### DISCUSSION THREAD

**Discussion by camillodev** [frontendbr/forum#1353] [Discussão]
Eu to querendo comprar um notebook bom, daquele dos sonhos mesmo. Então to disposto a pagar o preço deles se o custo beneficio for bom mesmo, e que dure uns 5 anos.
Só que sofro de indecisão cronica. To a uns dois meses pra decidir. 
Peso pra mim é um dos fatores determinantes, mas coloquei um pc de 1,8 kg na bolsa hj e tava super leve entao na lista abaixo, todos os selecionados tem no maximo 1,9kg
Eis minhas opções.

O obvio **Macbook pro de 13 ou 15 polegadas** com i7, ssd 256gb e 16gb de ram, 1.83 kg
o de 15 vem com placa de video de 4gb. Penso nele pelo poder de revenda de conseguir o que investi de volta msm dps de alguns anos de uso e pela fama mas não sou mto fa de apple. 
Medo pq ja vi mta gente reclamando da qualidade dos aparelhos da apple hoje em dia. 
**Preço: 1.800 dolares a 2400 dolares**
https://www.apple.com/shop/buy-mac/macbook-pro/15-inch-space-gray-2.2ghz-6-core-256gb#

Aqui começa os **2 em 1** que acredito que me vai ser mto util pq leio mto em tablet, e o meu atual ta bem ruim ja, e acredito que vai ajudar na hora de fazer rabisco frame e documentar. queria experimentar um 2 em 1 msm. 

Meu queridinho mas caro **Surface Book 2 15", o melhor 2 em 1 do mercado**
i7 8 geração, 16gb de ram, ssd 256gb, GTX 1060 6GB, 1,9kg
Caneta, 2 em 1, despluga e vira tablet
To muito tentado em comprar ele mas o pc debaixo tem as mesmas configurações (nao preciso de tanta placa de video), pela metade. Mas esse tem um design mto mais foda, e o fato de soltar me encanta bastante. enfim, é o que to apaixonado. Mas acho q ta mto salgado. 
**Preço: 2300 dolares**
https://www.microsoft.com/en-us/p/surface-book-2/8mcpzjjcc98c?activetab=pivot:techspecstab

**Samsung notebook pen 9 pro (aqui no brasil é samsung style s51)**
Essa versão é de 15 polegadas, 16gb ram, 256gb sdd, i7 8 geração, AMD Radeon 540 GDDR5 2GB, S pen, 2 em 1,
Preço absurdamente barato, design bem feio e modesto. 
Caneta meio pequena mas nada mto ruim. 
**Garantia Internacional** -- Os outros não sei se tem. 
**Preço: 1200 dolares (ta mto barato)**
https://www.samsung.com/us/computing/windows-laptops/notebook-series-9/notebook-9-pro-15---256gb-ssd--np940x5n-x01us/#specs

O que acham?
Vou pedir pra uma amiga trazer um desses para mim, ou ir mesmo no final do ano comprar la fora.

**Comment by vandaimer** [comment]
Olá @camillodev,

Também estou a procura de um notebook, com ótimo desempenho e leve, tem que ser leve, no meu caso é o que irá mais contar.

Enfim...Acredito que o **Samsung notebook pen 9 pro** é uma boa pedida.

Veja esse https://www.fastshop.com.br/web/p/d/UXX510URBQCNZ_PRD/notebook-asus-i7-8gb-1tb-15-6-pol-cinza-x510ur-bq292t-fast?utm_source=aff_zanox&utm_medium=aff&utm_campaign=zanox_deeplink&utm_content=2347880&zanpid=2521516255467643904&partner=parceiro-zanox

Em questão de hardware, talvez tenha diferença na placa de vídeo, e neste é HD, que da pra trocar por ssd fácil. Valor muito bom, leve e hardware foda.

**Comment by camillodev** [comment]
Esse asus e bom mas teria que colocar ssd e mais memoria. 
E intermediario ele, querendo algo mais avancado. Mas adoro asus

**Comment by luizbills** [comment]
Aqui no brasil custo-beneficio se chama Dell. Lenovo também tem bons notebooks hoje em dia.
Se for comprar outra marca, tenha certeza que a empresa oferece suporte técnico dentro do brasil.
Não recomendo notebook da sansung ou acer, dão muito defeito.

**Comment by andrelsa** [comment]
Nessa faixa de preço eu iria de notebook gamer, mas você prioriza que seja leve, então não entra no que você deseja. Mas nessa faixa de preço com boas configurações para durar 5 anos, no meu caso compraria um mac ou algum notebook gamer.

**Comment by bizarro** [comment]
@camillodev, se você vai comprar lá fora e no final do ano, várias empresas estão lançando notebooks com a GeForce RTX 20 Series. Se você não faz questão de que seja OS X, eu recomendaria algum desses aqui:

- [Razer Blade](https://www.razer.com/gaming-laptops/razer-blade).
- [Gigabyte AERO](https://www.gigabyte.com/Laptop/AERO-series).
- [MSI GS65](https://www.msi.com/Laptop/GS65-Stealth-8SX).

Todos eles são gamers, mas esses modelos parecem ser bem mais leves do que os que estão disponíveis aqui no Brazil.

**Comment by camillodev** [comment]
@lhbzr adorei as dicas mano. Vou dar uma olhada neles sim. Brigadão.

**Comment by ghost** [comment]
Recentemente decidi que ia trocar meu notebook e automaticamente lembrei dessa thread, então vou deixar aqui os critérios adotados e o modelo escolhido.

Pra mim, o notebook precisava ser pequeno, leve, porém potente e ter tela full HD. E custar no máximo uns R$ 4.500,00.

Durante as minhas buscas, encontrei um modelo que se adequa perfeitamente às minhas necessidades, que é o **Dell i14 7000**, mais especificamente o modelo **i14-7472-A20S**, e aqui deixo um vídeo curtinho que[ explica a diferença entra os 12 modelos](https://www.youtube.com/watch?v=hZEu88Kf1j8) dessa linha.

### Configuração:

- **Processador:** i7 8550U
- **RAM:** 8 GB DDR4 2400MHz (expansível até 16 GB)
- **Armazenamento:** 1 TB HD (Tem entrada pra SSD M.2)
- **Placa de vídeo:** MX 150 GDDR5 4 GB
- **Tela:** 14" IPS Truelife (1920 x 1080)
 
Ele ultrapassou um pouco minha expectativa de preço, mas achei que valia a pena. Além disso, dá pra ir melhorando em quantidade de memória RAM, tirar o HD e colocar um SSD ou SSHD na entrada SATA e um M.2, além de ser bem leve (1,54 kg).

Bom, fica aí minha contribuição que foi quase um jabá XD

**Comment by vandaimer** [comment]
Legal @GuimaraesJean.

Esse é menor e mais leve em relação ao que eu mostrei. Gostei mais deste.

Quero dar uma olhada se encontro um de 13", acho que consigo um mais leve e só não terei placa de vídeo dedicada.

Vlw.

**Comment by lu-zen** [comment]
Comprando fora eu iria de ThinkPad.

**Comment by vandaimer** [comment]
Verdade, esqueci do ThinkPad. Esse aqui me pareceu uma boa escolha.

https://www.lenovo.com/us/en/laptops/thinkpad/thinkpad-t-series/ThinkPad-T480s/p/22TP2TT480S

**Comment by longxia01** [comment]
Em termos de configuração da portátil, vê se este tópico te ajuda. 

[https://www.xp-pen.pt/forum-1411.html](https://www.xp-pen.pt/forum-1411.html)

**Comment by Pedrew25** [comment]
Tô querendo comprar um note bom ou mais ou menos, mas barato. No máximo 1.500. Pode ser novo ou usado. Alguma sugestão?
